### ISSUE TITLE
slow_conv3d_forward segfaults on CPU with very large SymInt convolution parameters

### ISSUE THREAD

**Issue by munianqianluo** [pytorch/pytorch#194016]
### Bug description

`torch.ops.aten.slow_conv3d_forward` segfaults on CPU when called with very large `SymInt` convolution parameters.

The reproducer uses a very large stride (`2^55`), large padding, and a `kernel_size` that is much larger than the spatial dimensions of the input. Instead of rejecting the invalid or unsupported parameter combination with a Python exception, the CPU process is terminated by `SIGSEGV`.

### Minimal reproduction

```python
import torch

x = torch.full(
    (2, 9, 8, 1, 2),
    3.40282e38,
    dtype=torch.float32,
    device="cpu",
)

w = torch.full(
    (1, 9, 9, 9, 1),
    2.718,
    dtype=torch.float32,
    device="cpu",
)

stride = 36028797018963968  # 2^55

torch.ops.aten.slow_conv3d_forward(
    x,
    w,
    [256, 256, 256],                 # kernel_size
    None,                            # bias
    [stride, stride, stride],        # stride
    [1000000, 1000000, 1000000],    # padding
)
```

### Observed behavior

With PyTorch `2.13.0+cpu`, the process is terminated by `SIGSEGV`.

When the reproducer was executed in a child process so that the termination signal could be captured, the result was:

```text
return code : -11
child killed by SIGSEGV (11)
```

No catchable Python exception is produced.

### Expected behavior

The operator should validate the convolution parameters and reject parameter combinations that cannot be processed safely with a catchable Python exception instead of terminating the process.

In particular, very large `SymInt` values used for `stride`, `padding`, or output-size calculations should not result in a native process crash.

### CUDA behavior

This operator cannot be directly compared with the CUDA backend in this PyTorch build.

With PyTorch `2.13.0+cu126`, the dispatcher reports that `aten::slow_conv3d_forward` has no CUDA kernel registered:

```text
CUDA kernel registered: False
```

Calling the operator with CUDA tensors therefore raises a `NotImplementedError` because the CUDA backend implementation is unavailable, rather than exercising the same kernel path.

### Environment

The CPU reproduction was performed with the official PyTorch `2.13.0+cpu` wheel in an isolated Python 3.10 Conda environment.

The test was run with `PYTHONPATH` and `LD_LIBRARY_PATH` unset and `PYTHONNOUSERSITE=1` to avoid loading packages from other PyTorch environments.

Full environment information is provided in the Versions section below.

### Versions

Collecting environment information...
PyTorch version: 2.13.0+cpu
Is debug build: False
CUDA used to build PyTorch: Could not collect
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.6 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0
Clang version: 10.0.0-4ubuntu1
CMake version: version 3.16.3
Libc version: glibc-2.31

Python version: 3.10.20 (main, Jun 11 2026, 15:17:37) [GCC 14.3.0] (64-bit runtime)
Python platform: Linux-5.15.0-139-generic-x86_64-with-glibc2.31
Is CUDA available: False
CUDA runtime version: 11.4.48
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration:
GPU 0: NVIDIA GeForce RTX 3090
GPU 1: NVIDIA GeForce RTX 3090

Nvidia driver version: 550.144.03
cuDNN version: Probably one of the following:
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_adv_train.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8.6.0
/usr/local/cuda-11.4/targets/x86_64-linux/lib/libcudnn_ops_train.so.8.6.0
Is XPU available: False
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: False
Caching allocator config: N/A

CPU:
Architecture:                         x86_64
CPU op-mode(s):                       32-bit, 64-bit
Byte Order:                           Little Endian
Address sizes:                        46 bits physical, 48 bits virtual
CPU(s):                               80
On-line CPU(s) list:                  0-79
Thread(s) per core:                   2
Core(s) per socket:                   20
Socket(s):                            2
NUMA node(s):                         2
Vendor ID:                            GenuineIntel
CPU family:                           6
Model:                                85
Model name:                           Intel(R) Xeon(R) Gold 5218R CPU @ 2.10GHz
Stepping:                             7
CPU MHz:                              800.000
CPU max MHz:                          4000.0000
CPU min MHz:                          800.0000
BogoMIPS:                             4200.00
Virtualization:                       VT-x
L1d cache:                            1.3 MiB
L1i cache:                            1.3 MiB
L2 cache:                             40 MiB
L3 cache:                             55 MiB
NUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78
NUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79
Vulnerability Gather data sampling:   Mitigation; Microcode
Vulnerability Itlb multihit:          KVM: Mitigation: VMX disabled
Vulnerability L1tf:                   Not affected
Vulnerability Mds:                    Not affected
Vulnerability Meltdown:               Not affected
Vulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable
Vulnerability Reg file data sampling: Not affected
Vulnerability Retbleed:               Mitigation; Enhanced IBRS
Vulnerability Spec rstack overflow:   Not affected
Vulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop
Vulnerability Srbds:                  Not affected
Vulnerability Tsx async abort:        Mitigation; TSX disabled
Flags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 cdp_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqmmpx rdt_a avx512f avx512dq rdseed adx smap clflushopt clwb intel_pt avx512cd avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts pku ospke avx512_vnni md_clear flush_l1d arch_capabilities

Versions of relevant libraries:
[pip3] mkl_fft==2.2.0
[pip3] mkl_random==1.4.1
[pip3] mkl-service==2.7.2
[pip3] numpy==2.2.5
[pip3] torch==2.13.0+cpu
[conda] Could not collect

**Comment by sohamdeshmukh-dev** [comment]
@munianqianluo can I get assigned this issue?

**Comment by soulitzer** [comment]
It is expected that non-public ops don't have all the error checks
